src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -126,6 +132,10 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(this.themeService.onDidColorThemeChange(this.updateTheme, this));
 		this.updateTheme(this.themeService.getColorTheme());
 		this.initActionBar();
+		if (this.contextKeyService.getContextKeyValue(localize("isDevelopmentContextKey", "isDevelopment")) &&
```

Comment:
Please remove the `localize` calls here. This is a static string value, so in non-english environments you wouldn't get the right value here. All context keys are non-localizable.